packages/gatsby-plugin-guess-js/README.md

```diff
@@ -47,3 +49,15 @@ module.exports = {
   ],
 }
 ```
+
+### Local development
+
+Using the plugin locally you'll need to authenticate the plugin with GA to gain access to your data. When building you'll be required to sign in to the organisation you'll be feeding the data from.
+
+### Production
+
+If deploying with a service such as netlify or heroku etc, you wont be able to grant access via Google Sign On. To get around this you'll need to generate a JWT for your GA account to pass to the plugin, this will enable the plugin to work in prod without any human interaction to authenticate permissions.
+
+## How to get a JWT token?
+
+[Here](https://2ality.com/2015/10/google-analytics-api.html)
```

Comment:
I feel like we should just copy and modify https://2ality.com/2015/10/google-analytics-api.html#unlocking-the-google-analytics-api in the readme and give credits to 2ality url.

Reply:
Another good point, I would probably be welcome to more suggestions as to how to do this in a simpler way (if possible?).

Maybe attaching something to a gatsby preBuild api to get a fresh JWT. My knowledge on JWT is a little limited but I would assume that this wouldn't be the best way as each JWT is valid until you make it non valid correct?